src/libraries/System.Net.Http/tests/PerformanceTests/HPackHuffmanBenchmark/HPackHuffmanBenchmark.csproj

```
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

```suggestion
    <StringResourcesPath>../../../src/Resources/Strings.resx</StringResourcesPath>
    <OutputType>Exe</OutputType>
```
Instead of Fakes/SR.cs. The same way it's solved in FunctionalTests:
https://github.com/dotnet/runtime/blob/3bcd31d5a1567fd022f41c375bb3953db94cc5dd/src/libraries/System.Net.Http/tests/FunctionalTests/System.Net.Http.Functional.Tests.csproj#L3-L3

---

works as magic :-)